amplify-graphql-types-generator depends on wrong version of core-js

https://github.com/aws-amplify/amplify-cli/blob/844051b260561af20c8eb2be559f5ae6694af4d7/packages/amplify-graphql-types-generator/package.json#L46

The functions in the file below rely on polyfils that no longer exists (have been moved) in core-js 3.

https://github.com/aws-amplify/amplify-cli/blob/844051b260561af20c8eb2be559f5ae6694af4d7/packages/amplify-graphql-types-generator/src/polyfills.js#L1-L3

Seems to have been around at least as long as this commit https://github.com/aws-amplify/amplify-cli/commit/fd692e885962810ed9e6295899e24903413fe184. I am unable to run `amplify init`.

```
covid-19-refugee on  feature/suggested-data-modal [⇕$] via ⬢ v14.2.0 took 2m 1s 
➜ amplify init           
Note: It is recommended to run this command from the root of your app directory
? Do you want to use an existing environment? Yes
? Choose the environment you would like to use: dev
? Choose your default editor: Visual Studio Code
Using default provider  awscloudformation
init failed
/Users/arlyon/.config/yarn/global/node_modules/amplify-graphql-types-generator/lib/polyfills.js:1
Error: Cannot find module 'core-js/fn/object/values'
Require stack:
- /Users/arlyon/.config/yarn/global/node_modules/amplify-graphql-types-generator/lib/polyfills.js
- /Users/arlyon/.config/yarn/global/node_modules/amplify-graphql-types-generator/lib/index.js
- /Users/arlyon/.config/yarn/global/node_modules/amplify-codegen/src/commands/types.js
- /Users/arlyon/.config/yarn/global/node_modules/amplify-codegen/src/index.js
- /Users/arlyon/.config/yarn/global/node_modules/amplify-provider-awscloudformation/lib/graphql-codegen.js
- /Users/arlyon/.config/yarn/global/node_modules/amplify-provider-awscloudformation/lib/push-resources.js
- /Users/arlyon/.config/yarn/global/node_modules/amplify-provider-awscloudformation/lib/amplify-service-migrate.js
- /Users/arlyon/.config/yarn/global/node_modules/amplify-provider-awscloudformation/lib/initializer.js
- /Users/arlyon/.config/yarn/global/node_modules/amplify-provider-awscloudformation/index.js
- /Users/arlyon/.config/yarn/global/node_modules/@aws-amplify/cli/lib/init-steps/s2-initProviders.js
- /Users/arlyon/.config/yarn/global/node_modules/@aws-amplify/cli/lib/commands/init.js
- /Users/arlyon/.config/yarn/global/node_modules/@aws-amplify/cli/lib/index.js
- /Users/arlyon/.config/yarn/global/node_modules/@aws-amplify/cli/bin/amplify
    at Object.<anonymous> (/Users/arlyon/.config/yarn/global/node_modules/amplify-graphql-types-generator/src/polyfills.js:1)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1196:10) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/arlyon/.config/yarn/global/node_modules/amplify-graphql-types-generator/lib/polyfills.js',
    '/Users/arlyon/.config/yarn/global/node_modules/amplify-graphql-types-generator/lib/index.js',
    '/Users/arlyon/.config/yarn/global/node_modules/amplify-codegen/src/commands/types.js',
    '/Users/arlyon/.config/yarn/global/node_modules/amplify-codegen/src/index.js',
    '/Users/arlyon/.config/yarn/global/node_modules/amplify-provider-awscloudformation/lib/graphql-codegen.js',
    '/Users/arlyon/.config/yarn/global/node_modules/amplify-provider-awscloudformation/lib/push-resources.js',
    '/Users/arlyon/.config/yarn/global/node_modules/amplify-provider-awscloudformation/lib/amplify-service-migrate.js',
    '/Users/arlyon/.config/yarn/global/node_modules/amplify-provider-awscloudformation/lib/initializer.js',
    '/Users/arlyon/.config/yarn/global/node_modules/amplify-provider-awscloudformation/index.js',
    '/Users/arlyon/.config/yarn/global/node_modules/@aws-amplify/cli/lib/init-steps/s2-initProviders.js',
    '/Users/arlyon/.config/yarn/global/node_modules/@aws-amplify/cli/lib/commands/init.js',
    '/Users/arlyon/.config/yarn/global/node_modules/@aws-amplify/cli/lib/index.js',
    '/Users/arlyon/.config/yarn/global/node_modules/@aws-amplify/cli/bin/amplify'
  ]
}
```

--- a/packages/amplify-graphql-types-generator/src/polyfills.js
+++ b/packages/amplify-graphql-types-generator/src/polyfills.js
@@ -1,3 +1,3 @@
-import 'core-js/fn/object/values';
-import 'core-js/fn/object/entries';
-import 'core-js/fn/array/flat-map';
+import 'core-js/stable/object/values';
+import 'core-js/stable/object/entries';
+import 'core-js/stable/array/flat-map';

